lib/utils.dart

```diff
@@ -75,4 +75,81 @@ abstract class CryptoUtils {
    */
   static Uint8List base64StringToBytes(String input) =>
       new Base64Decoder().convert(input);
+
+  static BigInt bytesToBigInt(List s) {
```

Comment:
Where did you find implementations of this methods??

Reply:
To be honest, I don't remember any more 🙈 

I remember I googled until I managed to solve the problem.. Now, googling again, I might have just cut out the parts of bignum that were relevant, and shoved it here. 

Anyway, since Dart 2.0 stable is out, this pull request might very well be obsolete. I was away too long and everything is on fire, but I will let you know as soon as I figure it out.

Reply:
With Dart 2 out, it's all but obsolete IMO. The methods should be updated to BigInt. Question is if BigInt [came out with byte conversion methods](https://github.com/dart-lang/sdk/issues/32803), but I doubt so..